extruct/uniform.py

```diff
@@ -23,6 +24,22 @@ def _umicrodata_microformat(extracted, schema_context):
     return res
 
 
+def _udublincore(extracted):
+    out = []
+    for obj in extracted:
+        context = obj.pop('namespaces', None)
```

Comment:
I think it is better to not modify the passed object here, and do a copy

Reply:
modified, thanks!

Reply:
by `list(extracted)` you're doing a shallow copy; changes in list elements still happen in-place

Reply:
Hey @joaquingx! Do you think you can address the comment above? It seems this is the only remaining issue, the PR has everything: a clean implementation, docs, tests, ..

Reply:
Sure ill work on that :+1:

Reply:
Done